toolkit/scripts/precache.mk

```diff
@@ -26,13 +26,15 @@ clean-precache:
 # it does, so add the phony target as a dependency to the flag file
 .PHONY: precache_always_run_phony
 pre-cache: $(STATUS_FLAGS_DIR)/precache.flag
-$(STATUS_FLAGS_DIR)/precache.flag: $(go-precacher) $(chroot_worker) $(rpms_snapshot) precache_always_run_phony 
+# If we set CUSTOM_PRECACHE_FILE, we want to use that instead of the default rpms_snapshot
+precache_input_snapshot = $(if $(CUSTOM_PRECACHE_FILE),$(CUSTOM_PRECACHE_FILE),$(rpms_snapshot))
```

Comment:
Please move it to `Makefile` and set it to a default value of `$(rpms_snapshot)`. We keep other input arguments modifiable by the user in that file. Afterwards we could also drop the "CUSTOM_" prefix as it's always going to be used with an option to be modified by the user. I'd maybe re-name it something along the lines of "PRECACHER_SNAPSHOT", so it's clearer what kind of file this may be.

Example:

```
PRECACHER_SNAPSHOT ?= $(rpms_snapshot)
```

The `precache_input_snapshot ` variable should be replacable with the new variable with that change as well.

Reply:
@PawelWMS Move to which `Makefile`? I see $(rpms_snapshot) is first defined in toolkit.mk and its default value depends on a bunch of other variables.